localstack/services/infra.py

```diff
@@ -373,6 +373,8 @@ def print_runtime_information(in_docker=False):
 
 
 def start_infra(asynchronous=False, apis=None):
+    config.dirs.mkdirs()
+
```

Comment:
Is this necessary? It seems to be done implicitly when loading the `config.py` for most cases.

Reply:
see comment above. i removed the implicit creation in config, which now requires it here, which is much clearer.